packages/manager/apps/pci-block-storage/src/pages/list/List.page.tsx

```diff
@@ -90,183 +66,184 @@ export default function ListingPage() {
+              links={CHANGELOG_LINKS}
+              chapters={CHANGELOG_CHAPTERS}
+            />
+          }
+        />
+      </div>
+      <OsdsDivider></OsdsDivider>
 
-        {hasMaintenance && (
-          <PciMaintenanceBanner
-            maintenanceURL={maintenanceURL}
-            productName={t('pci_projects_project_storages_blocks_title')}
-          />
-        )}
+      <PciAnnouncementBanner />
 
-        <PciAnnouncementBanner />
+      <Notifications />
 
-        <Notifications />
-        <div className="sm:flex items-center justify-between mt-4">
-          <OsdsButton
-            size={ODS_BUTTON_SIZE.sm}
-            variant={ODS_BUTTON_VARIANT.flat}
-            color={ODS_THEME_COLOR_INTENT.primary}
-            className="xs:mb-0.5 sm:mb-0"
-            onClick={() => {
-              clearNotifications();
-              navigate('./new');
-            }}
-          >
-            <OsdsIcon
-              size={ODS_ICON_SIZE.xs}
-              name={ODS_ICON_NAME.PLUS}
-              className="mr-4 bg-white"
-            />
-            {t('pci_projects_project_storages_blocks_add_label')}
-          </OsdsButton>
-          <div className="justify-between flex">
-            <OsdsSearchBar
-              className="w-[70%]"
-              value={searchField}
-              onOdsSearchSubmit={({ detail }) => {
-                setPagination({
-                  pageIndex: 0,
-                  pageSize: pagination.pageSize,
-                });
-                addFilter({
-                  key: 'name',
-                  value: detail.inputValue,
-                  comparator: FilterComparator.Includes,
-                  label: '',
-                });
-                setSearchField('');
-              }}
-            />
-            <OsdsPopover ref={filterPopoverRef}>
+      <div className="sm:flex items-center justify-between mt-4 min-h-[40px]">
+        <OsdsButton
+          size={ODS_BUTTON_SIZE.sm}
+          variant={ODS_BUTTON_VARIANT.flat}
+          color={ODS_THEME_COLOR_INTENT.primary}
+          className="xs:mb-0.5 sm:mb-0"
+          onClick={() => {
+            clearNotifications();
+            navigate('./new');
+          }}
+        >
+          <OsdsIcon
+            size={ODS_ICON_SIZE.xs}
+            name={ODS_ICON_NAME.PLUS}
+            className="mr-4 bg-white"
+          />
+          {t('pci_projects_project_storages_blocks_add_label')}
+        </OsdsButton>
+
+        <div className="justify-between flex gap-5 min-h-[40px] items-center">
+          <div className="flex items-center">
+            {isFetching ? (
+              <OsdsSpinner inline size={ODS_SPINNER_SIZE.md} />
+            ) : (
               <OsdsButton
-                slot="popover-trigger"
                 size={ODS_BUTTON_SIZE.sm}
                 color={ODS_THEME_COLOR_INTENT.primary}
                 variant={ODS_BUTTON_VARIANT.stroked}
+                onClick={() => refetch()}
```

Comment:
```suggestion
                onClick={refetch}
```

Reply:
It doesn't work since onClick sends a MouseClickEvent and refetch accepts an object with options